modules/cats/README.md

```diff
@@ -16,47 +16,44 @@ libraryDependencies += "com.github.pureconfig" %% "pureconfig-cats" % "0.9.1"
 
 ### Reading cats data structures from a config
 
-To load a `NonEmptyList[Int]` into a configuration, we need a class to hold our configuration:
+The following cats data structures are supported: 
+
+* `NonEmptyList`, `NonEmptyVector`, `NonEmptySet`
+* `NonEmptyMap[K, V]` only string keys are supported by default because it relies on `Map`.
+For a custom key you'll also have to provide an implicit on Ordering[K].
```

Comment:
Leftover `Ordering` reference. Also, we should mention that we also need instances for `Map[K, V]` in general case, otherwise it sounds like we only need an `Order`. Something like this:

> * `NonEmptyMap[K, V]` (from `ConfigReader[Map[K, V]]` and `Order[K]`).

Not sure it makes sense to mention that we only support strings here. That is a limitation on the instance for `Map` - let's not drag it around.

Reply:
Instead of specifying the constraints of reading to `Map`s, we can explain more generally that all of these data structures rely on the instances of their unrestricted (i.e. possibly empty) variants. It would be good for users that their custom collection readers, if any, affect the behavior of these too (it is a concern of special importance for maps because there are [configurable map instances](https://github.com/pureconfig/pureconfig/blob/master/core/src/main/scala/pureconfig/configurable/package.scala#L62)).